.github/scripts/scraper.py

```
@@ -11,145 +13,212 @@
-        
-        items = soup.find_all('item')
-        logger.info(f"見つかったアイテム数: {len(items)}")
+        """戻り値 (papers, fetch_failed)。"""
+        logger.info("Hugging Faceからの論文スクレイピングを開始します（公式API）")
+        r = self._request(_HF_API)
+        if r is None:
+            logger.error("Hugging Face API の取得に失敗しました（fetch_failed）")
+            return [], True
+        try:
+            data = r.json()
+        except Exception as ex:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)